lib/ts/recipe/passwordless/components/features/linkClickedScreen/index.tsx

```diff
@@ -118,15 +118,14 @@ const LinkClickedScreen: React.FC<PropType> = (props) => {
                 return Session.getInstanceOrThrow()
                     .validateGlobalClaimsAndHandleSuccessRedirection(
                         {
-                            rid: props.recipe.config.recipeId,
-                            successRedirectContext: {
-                                action: "SUCCESS",
-                                isNewPrimaryUser:
-                                    response.createdNewRecipeUser && response.user.loginMethods.length === 1,
-                                isNewRecipeUser: response.createdNewRecipeUser,
-                                redirectToPath: loginAttemptInfo?.redirectToPath,
-                            },
+                            action: "SUCCESS",
+                            isNewPrimaryUser: response.createdNewRecipeUser && response.user.loginMethods.length === 1,
+                            isNewRecipeUser: response.createdNewRecipeUser,
+                            isFirstFactor: true,
```

Comment:
shouldn't we set this based on the number of completed items in the session's mfa claim?